geometry_msgs/Pose new_world
---
bool success
string message